cime/src/drivers/mct/shr/seq_flds_mod.F90

```diff
@@ -1413,6 +1413,24 @@ subroutine seq_flds_set(nmlfile, ID, infodata)
     attname  = 'Fioi_meltw'
     call metadata_set(attname, longname, stdname, units)
 
+    ! Heat flux from melting icebergs
+    call seq_flds_add(i2x_fluxes,"PFioi_bergh")
+    call seq_flds_add(x2o_fluxes,"PFioi_bergh")
```

Comment:
You mean for these to be PF instead of F ?

Reply:
yes - the PF keeps them from getting multiplied by the ice fraction and passes directly through instead. Which is necessary because the iceberg fluxes are partly out past the seaice extent